test/msw-api/cli/init.test.ts

```diff
@@ -133,3 +133,37 @@ test('warns when current public directory does not match the saved directory fro
 
   await cleanup()
 })
+
+test('does not cause ESLint errors or warnings', async () => {
+  const { prepare, getPath, cleanup } = createTeardown(
+    './tmp/cli/init/eslint',
+    addFile('package.json', {
+      name: 'package-name',
+    }),
+    // intentionally degenerate config to cause a warning in the initial comment
+    addFile('.eslintrc.json', {
+      rules: {
+        'no-warning-comments': [
+          'warn',
+          { terms: ['mock'], location: 'anywhere' },
```

Comment:
So this triggers when a comment block has a "mock" phrase?

I was thinking maybe we can design these contradictory rules to stay agnostic (as possible) of the script's content? For example, forbidding the usage of `self`, which we cannot drop in a worker script.

Reply:
Depends on how thoroughly you want to test _this_ change of moving the `/* eslint-ignore */` line to the top; if our contradictory rules don't target the comment at all, this test will pass even without that change!

Reply:
You make a point. Let's keep the rule as-is now, as it indeed targets the introduced change closely. We can always iterate on this in the future if we see any problems (although it's unlikely).